packages/grafana-i18n/src/i18n.tsx

```diff
@@ -140,13 +152,17 @@ export function addResourceBundle(language: string, namespace: string, resource:
   getI18nInstance().addResourceBundle(language, namespace, resource, undefined, true);
 }
 
+function logWarning(entity: string) {
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
FWIW, I feel like this message should fail tests, as it indicates that you might be testing in a way that doesn't match reality. Then we can add test setup that makes sure it's initialised appropriately. I'm mainly thinking of trying to catch unintentional early calls to `t` in the tests, as when marking up parts of https://github.com/grafana/grafana/pull/105072, there are places where functions are called earlier than should be

Reply:
Fixed 👍